### PR TITLE
Fix the Issues #43

### PR DESCRIPTION
**Pull request by linsir**
Debian 9 + Python 2.7.13 + pymysql 0.7.12

### CODE REVIEW

**Comment by linsir** [comment]
Sorry, pymysql 0.7.11

File: torndb.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ class Connection(object):
     MySQLdb version >= 1.2.5 and MySQL version > 5.1.12.
     """
     def __init__(self, host, database, user=None, password=None,
-                 max_idle_time=7 * 3600, connect_timeout=0,
+                 max_idle_time=7 * 3600, connect_timeout=10,
```
Comment (bdarnell) [inline torndb.py]: Why did you change the connect timeout?

Comment (linsir) [reply torndb.py]: I remember it does not work because of the `connect_timeout=0`, so I change it , But I test it just now(`connect_timeout=0`), it works.
